.gitattributes

```diff
@@ -0,0 +1,2 @@
+Pipfile       text eol=lf
+Pipfile.lock  text eol=lf
```

Comment:
Probably not required, as the whole codebase is LF anyway.

Reply:
For me it was required, because git changed it on upload. It is crucial, that the Pipfile does not get changed, because the file hash of the Pipfile is stored in Pipfile.lock. When deploying pipenv with `--deploy` the hash is checked and the deployment fails on missmatch

Reply:
Sounds like you have set `autocrlf = true` in your .gitconfig. In my experience, that's only causing trouble, and modern text editors work just fine with any line endings you prefer. Just the thought of my VCS implicitly changing the code I commit! 😬

Reply:
I need to check that. :sweat_smile: